fiftyone/utils/video.py

```diff
@@ -903,12 +924,19 @@ def _transform_video(
                 ffmpeg.run(inpath, outpath, verbose=verbose)
 
             did_transform = True
-        elif diff_path:
+        elif inpath != outpath:
             etau.copy_file(inpath, outpath)
 
-        if delete_original and diff_path:
+        if delete_original and inpath != outpath:
             etau.delete_file(inpath)
     except Exception as e:
```

Comment:
@brimoor Hey Brian in testing it seems that Ctrl-C (or "stop kernel" in Jupyter) is not getting caught, maybe bc eg KeyboardInterrupt derives from BaseException and not Exception?